plugins/zscaler/icon_zscaler/util/helpers.py

```diff
@@ -0,0 +1,121 @@
+from insightconnect_plugin_runtime.exceptions import PluginException
+from re import sub, match, split
+from typing import Union
+from icon_zscaler.util.constants import Assistance, Cause
+
+_SNAKE_CASE_REGEX = r"\b[a-z0-9]+(_[a-z0-9]+)*\b"
+_CAMEL_CASE_REGEX = r"\b[a-z0-9]+([A-Z][a-z]+[0-9]*)*\b"
+_PASCAL_CASE_REGEX = r"\b[A-Z][a-z]+[0-9]*([A-Z][a-z]+[0-9]*)*\b"
+_CAMEL_CASE_ACRONYM_REGEX = r"\b[a-z0-9]+([A-Z]+[0-9]*)*\b"
+
+
+def clean_dict(dictionary: dict) -> dict:
```

Comment:
General typing is fine, but I suggest using more structured typing 😄 
`from typing import Any, Dict`
```suggestion
def clean_dict(dictionary: Dict[str, Any]) -> Dict[str, Any]:
```

Reply:
Updated. Thanks for your suggestion.